src/utils.js

```diff
@@ -13,8 +13,17 @@ export const getComponentVariant = (theme, componentName, variant) => {
 
 export const getComponentStyle = componentName => themeGet(`components.${componentName}.style`);
 
-export const getComponentClassName = ({ className, theme: { classPrefix }, variant }, name) =>
-  `${className || ''} ${classPrefix}-${name} ${variant ? `${classPrefix}-${name}-${variant}` : ''}`.trim();
+const getComponentClassName = flow(
```

Comment:
I like the fp approach. Flow is great for running a series of transformations, but here we're only running one, which could be rewritten as `'hey im a string'.split(' ').filter(Boolean).join(' ')`. I also don't quite see what this is doing that `trim()` doesn't.